.circleci/config.yml

```diff
@@ -17,14 +17,24 @@ jobs:
           name: Run npm install
           command: |
             npm install
+      - run:
+          name: verify error conformance
+          command: |
+            curl -sSL https://github.com/googleapis/gapic-config-validator/releases/download/v0.5.0/gapic-config-validator-0.5.0-linux-amd64.tar.gz > config-validator.tar
```

Comment:
it's `.tar.gz`, not `.tar` - please fix the name here after `>` and in the next line. Also, the extract command should be `tar xzf` (with `z`). I wonder why it works without `z`?

Reply:
I must have `xvf` the `tar.gz`and it gives error, so I renamed to `tar.gz` to `tar` then it worked.

from online 
```
-x --extract = extract files from an archive
-v, --verbose = verbosely list files processed
-z, --gzip = gzipped files eg. for tar.gz packages
-f, --file ARCHIVE = use archive file or device ARCHIVE
```